test-suite/README.md

```diff
@@ -0,0 +1,18 @@
+# Coq Test Suite
+
+The test suite can be run from the root directory by `make test-suite`.
```

Comment:
maybe "Coq root directory", since you don't mean /.

Reply:
Running `make test-suite` from the Coq root does a clean step first, so if you've already run it, then change something, you'll have to do a lot of work again.

If you run `make` from the test-suite directory, there's no clean step. You can also provide one of the directories, like `output`, there, to just build the logs for that directory, although you don't get the summary in that case.

Maybe some of this should be mentioned.

Reply:
Added.